### PR TITLE
refactor queries for wellbore, data objects

### LINKED ISSUE #473: 👉 Refactor query code for WELLBORE
**👉 Is your feature request related to a problem? Please describe.**
Related to #350
Sub-task for #350 

**Describe the solution you'd like**
In this task, we do refactor query code for WELLBORE

### PR DESCRIPTION
**Pull request by kdi-chandra**
# Request Template Witsml Explorer

## Fixes
This pull request fixes #473, #474, #475, #476, #477 

## Description

Refactoring the code especially Query package and change in related naming convention  

## Type of change

* Refactor

## Impacted Areas in Application

* Workers, Queries

## Checklist:

*Communication*
* [x] PR is related to an issue
* [ ] I have made corresponding changes to the documentation

*Code quality*
* [x] Code follows the style guidelines
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] Existing tests pass
* [ ] New code is covered by passing tests

## Further comments
Code is refactored and major functionality is tested and working as expected. 
All existing test cases are passed

### CODE REVIEW

File: Src/WitsmlExplorer.Api/Query/WellboreQueries.cs [hunk 1]
```diff
@@ -63,7 +64,43 @@ public static WitsmlWellbores UpdateWellboreQuery(string wellUid, string wellbor
             };
         }
 
-        public static WitsmlWellbores DeleteWellboreQuery(string wellUid, string wellboreUid)
+        public static WitsmlWellbores CreateWitsmlWellbore(Wellbore wellbore)
+        {
+            if (wellbore.WellboreParentUid != "")
```
Comment (lindsve) [inline Src/WitsmlExplorer.Api/Query/WellboreQueries.cs]: `if (!string.IsNullOrEmpty(wellbore.WellboreParentUid))`

**Review by lindsve — COMMENTED** [review]
Looks good to me. Just a small suggestion in a comment

*Reviewer checklist*
- [x] Read issue and PR
- [ ] Pulled branch and manually tested
- [ ] Verify that PR resolves issue
- [x] Reviewed the code

File: Src/WitsmlExplorer.Api/Query/WellboreQueries.cs (hunk 1 above)
Comment (kdi-chandra) [reply Src/WitsmlExplorer.Api/Query/WellboreQueries.cs]: Incorporated review comments

**Review by steinsiv — APPROVED** [review]
Tested for Wellbore, Logs, Trajectories, Messages, Rigs and Trajectorystations locally. Looks good 👍 . I was unable to test Risk object 
- [x] Read issue and PR
- [x] Pulled branch and manually tested
- [x] Verify that PR resolves issue #473 #474 #476 #477
- [x] Reviewed the code

**Pull request merged**